### ISSUE TITLE
Adjacent Links go to the Same URL for Thumbnail and Work

### ISSUE THREAD

**Issue by mtribone** [samvera-labs/nurax-pre2023#129]
### Descriptive summary

When viewing the works dashboard that contains my list of works, the thumbnail image and the title of the work both have hyperlinks that go the same URL. We can eliminate the redundant adjacent links by moving the thumbnail into the same link for the work title and style with CSS.

### Rationale

When adjacent links go to the same location (such as a linked product image and an adjacent linked product name that go to the same product page) this results in additional navigation and repetition for keyboard and screen reader users.

### Expected behavior

Expect thumbnail and title of the work to be within the same hyperlink.

### Actual behavior

By using Bootstrap's media object _inside_ of a table cell, we create additional complexity by separately linking the thumbnail and work title. 

### Steps to reproduce the behavior

1. Go to user dashboard
1. Select works from left-hand side
1. inspect listing of works or use WAVE accessibility toolbar
1. View the code

### Related work

https://github.com/samvera/sufia/issues/1276

**Comment by mtribone** [comment]
![screen shot 2017-10-31 at 10 00 10 am](https://user-images.githubusercontent.com/4163828/32227992-455970f8-be23-11e7-9386-527aa7ab9648.png)

**Comment by mjgiarlo** [comment]
@mtribone This change will likely need to be made in a number of related partials. Instead of combining the two into a single anchor, would it be defensible to remove one of the links (perhaps the thumbnail)?

**Comment by mjgiarlo** [comment]
Yes, that is true. If we make this change in one partial, I think we'd want to do it in all. I'm definitely not in the best position to say what the best UX here is! I defer to you and others.

**Comment by no-reply** [comment]
Based on my reading of #158, I believe this ticket has been addressed and can be closed.

**Comment by no-reply** [comment]
uggghhh... nevermind. It looks to me like the link was just removed from the thumbnail, which doesn't necessitate the change causing #158. Reopening until someone can revisit this from a directed UX standpoint.
